terra-jupyter-aou/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.1.7 - 2021-06-30
```

Comment:
Please also update config/conf.json (example PR: https://github.com/DataBiosphere/terra-docker/pull/212/files)

Reply:
conf.json was changed in 045a139, looks correct to me, but please let me know if I am misunderstanding you

Reply:
Sorry, I missed it somehow in the PR - lgtm